package.json

```diff
@@ -34,8 +34,8 @@
     "can-attribute-encoder": "^0.3.0",
     "can-compute": "^3.3.1",
     "can-event": "^3.5.0",
-    "can-event-dom-enter": "^1.0.2",
-    "can-event-dom-radiochange": "^1.0.2",
+    "can-event-dom-enter": "^1.0.4",
+    "can-event-dom-radiochange": "^1.0.4",
```

Comment:
Hi @roemhildtg, I would not expect it to be necessary for us to update these ranges… what does this fix?